src/components/Logo/Logo.tsx

```diff
@@ -31,17 +31,13 @@ export const Logo: React.FC<
 }) => {
     const hasWrapper = typeof wrapper === 'function';
 
-    let buttonIcon;
-
-    if (iconSrc) {
-        buttonIcon = (
-            <Button.Icon className={iconClassName}>
-                <img alt="logo icon" src={iconSrc} width={iconSize} height={iconSize} />
-            </Button.Icon>
-        );
-    } else if (icon) {
-        buttonIcon = <Icon data={icon} size={iconSize} className={iconClassName} />;
-    }
+    const buttonIcon = iconSrc ? (
+        <Button.Icon className={iconClassName}>
+            <img alt="logo icon" src={iconSrc} width={iconSize} height={iconSize} />
+        </Button.Icon>
+    ) : (
+        icon && <Icon data={icon} size={iconSize} className={iconClassName} />
+    );
```

Comment:
Nested condition in ternary operator is difficult for understanding
I prefer to keep old code with `if..else if`

Reply:
fixed